hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -827,79 +927,52 @@ logic is powered by VCAON.
 ## Core Power Control and Indication
 
 VCMAIN is the only supply that can be directly influenced by OpenTitan.
-The power manager can request VCMAIN to shutdown through main\_pd\_n.
-The state of VCMAIN is reflected by the vcmain\_pok\_o signal.
+The power manager can request VCMAIN to shutdown through main_pd_ni. The
+state of VCMAIN is reflected by the vcmain_pok_o signal.
 
 ## IO Power Indication
 
-IO power state is reflected to OpenTitan by vioa\_pok\_o and
-viob\_pok\_o signals
+IO power state is reflected to OpenTitan by vioa_pok_o and viob_pok_o
+signals
 
 ## Main (VCC) Power Detection and Flash Protection
 
-On VCC power-down detection, ‘flash\_power\_ready\_h\_o’,
-‘vcaon\_pok\_o’ and ‘vcmain\_pok\_o’ are immediately negated. This
-means that negation of the VCC supply always triggers the flash
+On VCC power-down detection, ‘flash_power_ready_h\_o’, is immediately
+negated. In addition, SYS clock, IO clock and USB clock are stopped.
+This means that negation of the VCC supply always triggers the flash
 brown-out (BOR) protection circuitry.
 
-When entering deep-sleep mode, ‘flash\_power\_down\_h\_o’ is asserted
+When entering deep-sleep mode, ‘flash_power_down_h\_o’ is asserted

```

Comment:
I don't think we need to use backslashes in these signal names? 
the back slashes have been removed in other places as well in this PR...